index.tests.js

```diff
@@ -264,3 +264,85 @@ describe('unknown app', () => {
     expect(logger.mock.calls).toEqual([['GET', '    ', '/test']]);
   });
 });
+
+describe('new features', () => {
```

Comment:
Can we rename this describe block to encompass what the tests cover. i.e `"path styles"`

```suggestion
describe('path styles', () => {
```